### PR TITLE
feat: add role created / deleted / updated events

### PR DESCRIPTION
**Pull request by yusdacra**

### CODE REVIEW

File: chat/v1/chat.proto [hunk 1]
```diff
@@ -37,7 +37,7 @@ service ChatService {
     option (harmonytypes.v1.metadata).requires_authentication = true;
   }
 
-  rpc GetGuildList(GetGuildListRequest) returns (GetGuildListResponse) {
+  rpc GetGuildList(google.protobuf.empty) returns (GetGuildListResponse) {
```
Comment (pontaoski) [inline chat/v1/chat.proto:40]: unrelated change?

Comment (yusdacra) [reply chat/v1/chat.proto:40]: Yeah let me open another PR for that

**Review by pontaoski — APPROVED** [review]
